appgate/customloaders.py

```diff
@@ -24,12 +24,38 @@ class CustomLoader:
 class CustomAttribLoader(CustomLoader):
     loader: Callable[[Any], Any] = attrib()
     field: str = attrib()
+    load_external: bool = attrib()
+
+    def load(self, values: AttributesDict) -> AttributesDict:
```

Comment:
is this called at all? Or is it supposed to be implemented in classes inheriting this?

Reply:
Yes, this is the loader used by the PasswordAttribMaker. Had to put in this `if self.load_external` to be able to handle both loading external secrets and decrypting k8s secret passwords.